internal/server/statvar_hierarchy_util.go

```diff
@@ -48,6 +48,8 @@ type SearchIndex struct {
 // PVs to a number greater than max number of PVs for a human curated stat var.
 const nonHumanCuratedNumPv = 30
 
+var blocklistedSvgIds = []string{"dc/g/Person_EmploymentStatus", "dc/g/Establishment", "dc/g/Person_Industry"}
```

Comment:
please add a comment that this should be synced with website repo

Reply:
done